### PR TITLE
Add install scripts

### PR DESCRIPTION
**Pull request by martinbonnin**

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+if [ -z "$JAVA_BIN" ]
+then
+    echo "no java executable found. Please set your JAVA_HOME"
+fi
+
+
+JAR_FILE=$(mktemp)
+
+curl -s "https://dl.bintray.com/dailymotion/com.dailymotion.kinta/com/dailymotion/kinta/kinta-cli/0.1.0/kinta-0.1.0.jar" > $JAR_FILE
```
Comment (michgauz) [inline scripts/install.sh]: This should not be something like : 
https://dl.bintray.com/dailymotion/com.dailymotion.kinta/com/dailymotion/kinta/kinta-cli/$v/kinta-cli-$v.jar
?

Comment (martinbonnin) [reply scripts/install.sh]: I wish but it's not easy to get the latest version from shell unfortunately

Comment (martinbonnin) [reply scripts/install.sh]: This should be better now that I deploy the last zip to https://dailymotion.github.io/kinta/kinta.zip

File: build.gradle.kts [hunk 2]
```diff
@@ -134,4 +134,13 @@ fun Project.configureMavenPublish() {
...
         }
     }
+}
+
+apply(from = "docs.gradle.kts")
+
+tasks.register("deployDocsIfNeeded") {
+    val branch = System.getenv("TRAVIS_BRANCH")
+    if (branch == "master") {
+        dependsOn("deployDocs")
```
Comment (michgauz) [inline build.gradle.kts]: Should we deploy docs on a PR build too ?
With this impl, it will

File: kinta-cli/build.gradle.kts [hunk 3]
```diff
@@ -29,20 +29,7 @@ dependencies {
...
-         * we remove INDEX.LIST else java does not find the MainClass
-         * since we remove INDEX.LIST, it also looks like we need to remove signatures too. Not 100% sure why
-         */
-        exclude("META-INF/*.RSA", "META-INF/*.SF", "META-INF/*.DSA", "META-INF/INDEX.LIST", "META-INF/*.kotlin_module")
-    }
-}
-
 application {
     mainClassName = "com.dailymotion.kinta.MainKt"
+    applicationName = "kinta"
```
Comment (michgauz) [inline kinta-cli/build.gradle.kts]: The generated jar file path will still have the same name ? kinta-cli.jar

File: build.gradle.kts (hunk 2 above)
Comment (martinbonnin) [reply build.gradle.kts]: Thanks, fixed.

File: kinta-cli/build.gradle.kts (hunk 3 above)
Comment (martinbonnin) [reply kinta-cli/build.gradle.kts]: The generated jar name will change but ultimately, we shouldn't download the jar directly but let install.sh do it.

Comment (michgauz) [reply kinta-cli/build.gradle.kts]: 👍

**Pull request merged**